extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -71,7 +71,8 @@ static public function isCorrectPageType() {
 		if ( $wg->Title->exists() &&
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1067)